jasper.proto

```diff
@@ -5,6 +5,25 @@ option go_package = "internal";
 
 import "google/protobuf/empty.proto";
 
+message Logger {
+    LogType log_type = 1;
+}
+
+message OutputOptions {
+  repeated Logger loggers = 1;
+}
```

Comment:
Won't we need to specify other options?

Reply:
I added the other options. Should the LogOptions also be exported?